src/pages/contact/success.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import Layout from '../../components/layout'
+
+export default () => (
+  <Layout title="Thank you" narrow textHeavy>
+    <h2>Thank you for contacting us!</h2>
+    <p>We will try our best to get in touch shortly.</p>
```

Comment:
Our accessibility statement provides an explicit timeline: `We try to respond to feedback within 5 business days, and to propose a solution within 10 business days.` I think it makes sense to put that here for the accessibility form.

Reply:
Makes sense. Do we think that copy will change across form types? (Accessibility, volunteering, general feedback, etc. etc.) Right now it's one success page, so if the copy changes from form to form, we'll have to pass the form type in, right?

Reply:
Let's not get into a dynamic success page, this page should be used by all the forms. How about we add that timeline to the preamble of `/src/pages/contact/accessibility.js`